src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx

```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
 9. Optional: Set [**Violation time limit** for violations](#violation-time-limit) (this defaults to 24 hours).
 10. Select **Create**.
 
-## Create alert conditions for infrastructure integrations [#integration]
-
-To create an alert condition for your infrastructure integrations, use any of these options:
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "250px" }}>
-        UI options
-      </th>
-
-      <th>
-        Alert conditions for infrastructure integrations
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        From **Alerts & AI**
-      </td>
-
-      <td>
-        Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
+<Callout variant="important">
+  If New Relic hasn't received a Cloud Integration service's attribute in the past 60 minutes, then it won't be selectable for alerting in the UI. You can use the [API](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/rest-api-calls-new-relic-infrastructure-alerts/) to create an alert condition for silent attributes.  
+</Callout>
 
-        OR
+## Other infrastructure alert condition methods [#methods]
+
+You can also use these other methods to create an infrastructure alert condition:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="alerts-ui"
+    title="Use the Alerts UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
```

Comment:
I don't have a test account to look at policy setup in UI but am assuming "product" is still the right word/ field name?

Reply:
Yup, it's still "product" in the UI